### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ironroggers**
Improved text readablity.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@vrothberg @mtrmac @TomSweeneyRedHat @QiWang19 PTAL

File: README.md [hunk 1]
```diff
@@ -23,22 +23,22 @@ Skopeo works with API V2 container image registries such as [docker.io](https://
 
  Skopeo operates on the following image and repository types:
 
- * containers-storage:docker-reference
+ * <i>containers-storage:docker-reference</i><br>
```
Comment (TomSweeneyRedHat) [inline README.md:26]: I like the end result, but I'd rather not add html.  Could you change this to the md equivalents please?

```suggestion
 * **containers-storage:docker-reference**  
```
two astericks on either side of the word, then two white spaces at the end of the line to force `<br>`

Comment (TomSweeneyRedHat) [reply README.md:26]: and ditto below.

**Comment by TomSweeneyRedHat** [comment]
@ironroggers and I rudely forgot to not thank you for the PR, so THANK YOU!

**Review by mtrmac — COMMENTED** [review]
If the intent is to give the text formatting some semantic significance, the literal parts of the syntax should use the corresponding Markdown literal syntax, and the variables should use something else (probably italic).

**Comment by ironroggers** [comment]
> @ironroggers and I rudely forgot to not thank you for the PR, so THANK YOU!

Welcome!!!

File: README.md [hunk 2]
```diff
@@ -130,19 +130,19 @@ $ skopeo inspect docker://registry.fedoraproject.org/fedora:latest | jq '.Digest
 ## Copying images
 
 `skopeo` can copy container images between various storage mechanisms, including:
-* Container registries
+* **Container registries**
```
Comment (mtrmac) [inline README.md:133]: Is there a semantic point to this? it’s just a list.

Comment (ironroggers) [reply README.md:133]: The main logic is to have a blog or website look better with perfect spacings and maintaining a good header-text ratio, which is what I tried. Is it good??

Comment (mtrmac) [reply README.md:133]: I mean, we are not in control of the CSS on this site anyway (and most of us probably wouldn’t _want_ to tinker with it if we could); so the way this content gets longevity and resilience to (GitHub’s and others’) CSS changes is by focusing on the semantics.

Comment (ironroggers) [reply README.md:133]: Okay!!! I got the point. My mistake

**Comment by TomSweeneyRedHat** [comment]
@mtrmac my suggestion would keep the formatting as @ironroggers had it in the html initially, i.e. a bulllet and italic for `containers-storage:docker-reference` followed by a line break and the description.  

Personally I found it nicer/easier to read, YMMV.

@ironroggers did you mean to close this?

**Comment by mtrmac** [comment]
Reformatting _that list of examples_ would make sense — but to something like `containers-storage:`_docker-reference_ , actually conveying something about the syntax.

Reformatting the list of destinations in the `skopeo copy` — really font weights are a negligible problem compared to the relative placement of that section and its place in the logic of the page, so optimizing font weights before wording/placement/structure would be at the very least premature optimization.

**Pull request closed without merging**